exotica_core/include/exotica_core/problems/bounded_end_pose_problem.h

```diff
@@ -33,10 +33,10 @@
 #ifndef BOUNDEDENDPOSEPROBLEM_H_
```

Comment:
None of the files have the header guards fixed. In other packages we now adopted `EXOTICA_PACKAGE_NAME_HEADER_NAME_H_`, e.g., `EXOTICA_AICO_SOLVER_MATH_OPERATIONS_H_`